projects/observability/src/shared/components/status-display/status-display.component.scss

```diff
@@ -0,0 +1,12 @@
+@import 'mixins';
+
+.status {
+  display: flex;
+  align-items: center;
```

Comment:
nit: you can use display-flex(column) from mixins here

Reply:
why? But it is not a column flow?

Reply:
oh sorry, my mistake, didn't see that